docs/parameters/summary_log.md

```diff
@@ -8,7 +8,13 @@ description: Return a summary of the parameters being set.
 
 This function returns a string that can be logged to the terminal, summarizing the parameters provided to the pipeline.
 
-It only returns the provided parameters that are **different** to the schema default values.
+!!! note
+
+    The summary only returns the provided parameters that are **different** to the schema default values.
+
+    This is because pipelines often have a very long list of parameters.
+    If we print all of them, it's difficult to see which ones were customised.
+    That's what users usually care about: for example, did I make a typo, was my config resolved properly, etc.
```

Comment:
```suggestion
The summary prioritizes displaying only the parameters that deviate from the default schema values. This is to streamline the extensive parameter lists often associated with pipelines, and highlight the customized elements. This feature is essential for users to verify their configurations, like checking for typos or confirming proper resolution, without wading through an array of default settings.
```

Reply:
@mirpedrol did you check the rendering on this before applying? I think the indentation was lost, which will have broken the admonition box.

Reply:
I didn't, will fix it! 😄